pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go

```diff
@@ -34,6 +34,7 @@ type ExperimentDef struct {
 	Command []string `json:"command"`
 	// Defines arguments to executor's entrypoint command
 	Args []string `json:"args"`
+	//Litmusbook string   `json:"litmusbook"`
```

Comment:
Is this comment required?

Reply:
Litmusbook section is added by me in clientset.But after that I found that it is not a required field.I will remove this